.changeset/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
```

Comment:
minor: use the version matching what we installed https://github.com/formspree/formspree-js/blob/3d0fe024702e86bede662b3997a8ad33630f7fe4/yarn.lock#L1220